pkg/virt-api/webhooks/validating-webhook/admitters/migration-create-admitter_test.go

```diff
@@ -34,11 +34,49 @@ import (
 	kubevirtfake "kubevirt.io/client-go/kubevirt/fake"
 
 	"kubevirt.io/kubevirt/pkg/libvmi"
+	"kubevirt.io/kubevirt/pkg/testutils"
 	"kubevirt.io/kubevirt/pkg/virt-api/webhooks"
 	"kubevirt.io/kubevirt/pkg/virt-api/webhooks/validating-webhook/admitters"
+	"kubevirt.io/kubevirt/pkg/virt-config/featuregate"
 )
 
 var _ = Describe("Validating MigrationCreate Admitter", func() {
+	const (
+		vmiRefName        = "vmiName"
+		testMigrationName = "testMigration"
+	)
+
+	kv := &v1.KubeVirt{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "kubevirt",
+			Namespace: "kubevirt",
+		},
+		Spec: v1.KubeVirtSpec{
+			Configuration: v1.KubeVirtConfiguration{
+				DeveloperConfiguration: &v1.DeveloperConfiguration{},
+			},
+		},
+		Status: v1.KubeVirtStatus{
+			Phase:               v1.KubeVirtPhaseDeploying,
+			DefaultArchitecture: "amd64",
+		},
+	}
+	config, _, kvStore := testutils.NewFakeClusterConfigUsingKV(kv)
+
+	enableFeatureGate := func(featureGate string) {
```

Comment:
That'll be the 12th `enableFeatureGate` unit test function... Maybe we can define it in a more central location?

Reply:
I thought it was strange it was not a central function either. I figured it had something to do with the fact that they were all scoped inside their particular unit test Describe, and there was a reason it was not centralized. I have no issue with doing that, but it would make this PR change a bunch of unrelated tests.